public/static/docs/user-guide/contributing/docs.md

```diff
@@ -134,4 +152,4 @@ pre-commit hook that is integrated when `yarn` installs the project dependencies
 
 > Check out the `.md` source code of any command reference to get a better idea,
 > for example in
-> [this very file](https://raw.githubusercontent.com/iterative/dvc.org/master/static/docs/user-guide/contributing/docs.md).
+> [this very file](https://raw.githubusercontent.com/iterative/dvc.org/master/docs/user-guide/contributing/docs.md).
```

Comment:
this is wrong URL, right? Also, this same file have other links to `sidebar.json` (and other files?). I would check README as well.

Reply:
Yeah, thanks for catching this. Will fix.

Reply:
*these

Reply:
Fixed this one.

Reply:
> this same file have other links to sidebar.json (and other files?). I would check README as well

Double checked (and fixed) all `iterative/dvc.org` links throughout repo.